src/embedding_wrapper.c

```diff
@@ -26,35 +21,20 @@ JULIA_DEFINE_FAST_TLS()
 // Declare C prototype of a function defined in Julia
 int julia_main(jl_array_t*);
 
-// main function (windows UTF16 -> UTF8 argument conversion code copied from julia's ui/repl.c)
-int main(int argc, char *argv[])
-{
-    uv_setup_args(argc, argv); // no-op on Windows
-
-    // initialization
-    libsupport_init();
-
-    // Get the current exe path so we can compute a relative depot path
```

Comment:
This is actually not needed since the default DEPOT_PATH includes `share/julia`.

Reply:
Ah, but that only works if one hasn't mucked with the `JULIA_DEPOT_PATH` env var.